test/unit/alphabet/adaptation/char_test.cpp

```diff
@@ -13,14 +13,18 @@
 
 #include "../alphabet_test_template.hpp"
 #include "../alphabet_constexpr_test_template.hpp"
+#include "../semi_alphabet_test_template.hpp"
+#include "../semi_alphabet_constexpr_test_template.hpp"
 
 using namespace seqan3;
 
 // char32_t and wchar_t, too slow
```

Comment:
this comment suggest something which is not reflected in the code?

Reply:
I don't understand your issue?
As the comment says char32_t and wchar_t are too slow to test in the test template because iterating over every possible rank value takes for ever.

Reply:
ah true. then the comment can go :+1: